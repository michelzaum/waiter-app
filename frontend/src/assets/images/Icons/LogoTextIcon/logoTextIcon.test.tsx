import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { LogoTextIcon } from '.';

describe('LogoTextIcon', () => {
  it('Should render the LogoTextIcon', () => {
    //given
    const result = render(<LogoTextIcon />);

    // then
    expect(result).toMatchSnapshot();
  });
});
