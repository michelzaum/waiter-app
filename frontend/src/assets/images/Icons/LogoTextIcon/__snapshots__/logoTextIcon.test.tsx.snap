// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LogoTextIcon > Should render the LogoTextIcon 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        fill="none"
        height="29"
        viewBox="0 0 42 29"
        width="42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M15.212 24L12.404 13.272L9.596 24H5.012L0.644 6.768H5.084L7.412 17.04L10.148 6.768H14.732L17.372 17.088L19.724 6.768H24.164L19.772 24H15.212ZM28.2965 17.88H36.8405L37.2005 18.864H27.6725L28.2965 17.88ZM38.6645 24L32.4485 7.992L26.2085 24H25.0565L31.8485 6.768H33.1205L39.8645 24H38.6645Z"
          fill="#666666"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="29"
      viewBox="0 0 42 29"
      width="42"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M15.212 24L12.404 13.272L9.596 24H5.012L0.644 6.768H5.084L7.412 17.04L10.148 6.768H14.732L17.372 17.088L19.724 6.768H24.164L19.772 24H15.212ZM28.2965 17.88H36.8405L37.2005 18.864H27.6725L28.2965 17.88ZM38.6645 24L32.4485 7.992L26.2085 24H25.0565L31.8485 6.768H33.1205L39.8645 24H38.6645Z"
        fill="#666666"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
